services/organization.js

```diff
@@ -102,7 +103,25 @@ async function updateOrg(id,name) {
     }
 }
 
+async function getOrganization(token) {
+    let uid;
+    getAuth()
```

Comment:
I'm pretty sure that getAuth() return a promise and you can use await to handle that because you already declare async funtion @Kaigagi   `await getAuth()` correct me if I'm wrong ok 🤔🤔

Reply:
fixed in "token for organization" commit